test/Microsoft.Data.Analysis.Tests/DataFrameTests.cs

```diff
@@ -798,6 +811,169 @@ public void TestComputations()
             }
         }
 
+        [Fact]
+        public void TestComputationsIncludingDateTime()
+        {
+            DataFrame df = MakeDataFrameWithNumericStringAndDateTimeColumns(10);
+            df["Int"][0] = -10;
+            Assert.Equal(-10, df.Columns["Int"][0]);
+
+            DataFrameColumn absColumn = df.Columns["Int"].Abs();
+            Assert.Equal(10, absColumn[0]);
+            Assert.Equal(-10, df.Columns["Int"][0]);
+            df.Columns["Int"].Abs(true);
+            Assert.Equal(10, df.Columns["Int"][0]);
+
+            Assert.Throws<NotSupportedException>(() => df.Columns["Byte"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Byte"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Char"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Char"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Decimal"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Decimal"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Double"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Double"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Float"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Float"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Int"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Int"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Long"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Long"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Sbyte"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Sbyte"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Short"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Short"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Uint"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Uint"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Ulong"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Ulong"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Ushort"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["Ushort"].Any());
+            Assert.Throws<NotSupportedException>(() => df.Columns["DateTime"].All());
+            Assert.Throws<NotSupportedException>(() => df.Columns["DateTime"].Any());
+
+            // Test the computation results
+            var maxDate = DateTime.Today.AddDays(100);
```

Comment:
This is going to break tomorrow.

Reply:
Yup yup, got pulled into a meeting before I could push the last commit. Fixed now